GroupBundle/DataFixtures/MongoDB/LoadGroupDemoData.php

```diff
@@ -18,9 +22,31 @@ class LoadGroupDemoData extends AbstractLoadGroupData implements OrchestraProduc
     public function load(ObjectManager $manager)
     {
         $group2 = $this->generateGroup('Demo group', 'Demo group', 'Groupe de démo', 'site2', 'group2');
-        $group2->addRole(AdministrationPanelStrategy::ROLE_ACCESS_REDIRECTION);
         $manager->persist($group2);
 
+        $adminGroup = $this->generateGroup('Admin group', 'Admin group', 'Groupe admin');
+        $adminGroup->addRole(AdministrationPanelStrategy::ROLE_ACCESS_REMOVED_TRASHCAN);
+        $adminGroup->addRole(AdministrationPanelStrategy::ROLE_ACCESS_THEME);
```

Comment:
i can't use the setRoles because there are already roles to which i add other roles. if i use setRoles i will override the roles that already exist.